### PR TITLE
Feature/user forum backend

### PR DESCRIPTION
**Pull request by sergiynikon**
ALSO ADDED REPLY Logic

### CODE REVIEW

**Review by RomanNakonechnyi — APPROVED** [review]

**Review by AttFlederX — APPROVED** [review]

File: src/backend/LDW.Persistance/Context/ApplicationDbContext.cs [hunk 1]
```diff
@@ -15,8 +15,11 @@ public ApplicationDbContext(DbContextOptions<ApplicationDbContext> options)
         }
 
         public DbSet<DormitoryEntity> Dormitories { get; set; }
+		public DbSet<ForumSectionEntity> ForumSections { get; set; }
+		public DbSet<ForumThreadEntity> ForumThreads { get; set; }
+		public DbSet<ForumThreadReplyEntity> ForumThreadReplies { get; set; }
```
Comment (Meownjik) [inline src/backend/LDW.Persistance/Context/ApplicationDbContext.cs:20]: Where have the lines gone? (Куди все з'їхало?)

**Review by Meownjik — APPROVED** [review]

File: src/backend/LDW.Persistance/Context/ApplicationDbContext.cs (hunk 1 above)
Comment (sergiynikon) [reply src/backend/LDW.Persistance/Context/ApplicationDbContext.cs:20]: it's a github issue. In visual studio these lines are displayed normally with normal indentation

**Review by Meownjik — APPROVED** [review]
That's a bit annoying when you have 100500 files changed in one pull request

**Review by AttFlederX — CHANGES_REQUESTED** [review]
Fix foreign key issues before merging

**Comment by sergiynikon** [comment]
> Fix foreign key issues before merging

done

**Review by AttFlederX — APPROVED** [review]

**Pull request merged**